gpflow/optimizers/natgrad.py

```diff
@@ -132,21 +178,29 @@ def _natgrad_step(
             year={2018}
 
         In addition, for convenience with the rest of GPflow, this code computes ∂L/∂η using
-        the chain rule:
+        the chain rule (the following assumes a numerator layout where the gradient is a row
+        vector; note that TensorFlow actually returns a column vector), where L is the loss:
 
-        ∂L/∂η = (∂L / ∂[q_μ, q_sqrt])(∂[q_μ, q_sqrt] / ∂η)
+        ∂L/∂η = (∂L / ∂[q_mu, q_sqrt])(∂[q_mu, q_sqrt] / ∂η)
```

Comment:
Why is that changed?

Reply:
to be consistent with the code

Reply:
i think given this is the name of the variables we are using here, its kinda helpful to have it as q_mu. in general agree being against superfluous changes to docs for formatting but yeah

Reply:
you are against cleaning up the docs ? O.o

Reply:
I'm not against cleaning the docs. The cleaning could be done in different direction. I like q_μ more, and it doesn't contradicts the code.

Reply:
q_μ is consistent with neither paper notation (where it is just μ) nor the code (where it is q_mu), it took me extra mental effort to parse this as referring to q_mu...

Reply:
As you can see I had an opposite view. I would argue that it is better to keep consistency within one thing, rather trying to find consistency among everything. In this case, it was a documentation.

Reply:
Sorry, did I not change it consistently - did I miss any occurrences? what is it that you think it's not consistent with, now?